mypyc/irbuild/classdef.py

```diff
@@ -165,7 +165,8 @@ def allocate_class(builder: IRBuilder, cdef: ClassDef) -> Value:
     base_exprs = cdef.base_type_exprs + cdef.removed_base_type_exprs
     if base_exprs:
         bases = [builder.accept(x) for x in base_exprs]
-        tp_bases = builder.primitive_op(new_tuple_op, bases, cdef.line)
+        load_size_op = builder.add(LoadInt(len(bases), -1, c_pyssize_t_rprimitive))
+        tp_bases = builder.call_c(new_tuple_op, [load_size_op] + bases, cdef.line)
```

Comment:
Since these two lines are duplicated several times in this PR, it would be better to move these to a helper function and use it in all call sites (looks like there are 5 in this PR).